<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PyAList [YCST]</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="setting_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QLabel" name="setting_alist_label">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>30</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AList 路径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="setting_alist_input">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>30</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>请选择 AList 程序路径</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setting_alist_btn">
     <property name="geometry">
      <rect>
       <x>283</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="console_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>380</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>控制台</string>
    </property>
    <widget class="QPushButton" name="console_alist_btn_run">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>运行</string>
     </property>
    </widget>
    <widget class="QLabel" name="console_alist_status_img">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="console_alist_status_label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>35</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>状态：</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>20</width>
       <height>191</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="console_alist_btn_open">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="console_alist_btn_stop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>30</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QPushButton" name="console_alist_btn_admin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>100</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>查看默认用户</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="help_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>帮助</string>
    </property>
    <widget class="QPushButton" name="help_document">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AList 文档</string>
     </property>
     <property name="url" stdset="0">
      <string>https://alist.nn.ci/zh/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="help_download">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AList 下载</string>
     </property>
     <property name="url" stdset="0">
      <string>https://github.com/alist-org/alist/releases</string>
     </property>
    </widget>
    <widget class="QPushButton" name="help_add_storage">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加存储</string>
     </property>
     <property name="url" stdset="0">
      <string>https://alist.nn.ci/zh/guide/drivers/common.html</string>
     </property>
    </widget>
    <widget class="QPushButton" name="help_webdav">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>WebDAV</string>
     </property>
     <property name="url" stdset="0">
      <string>https://alist.nn.ci/zh/guide/webdav.html</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
